run.sh

```diff
@@ -35,7 +35,7 @@ if command -v hyperfine &>/dev/null; then
     HYPER=1
 fi
 
-time="$(which gtime || which time)"
+time="$(which gtime 2>/dev/null || which time)"
```

Comment:
```suggestion
time="$(which gtime 2>/dev/null || which time 2>/dev/null)"
if [[ -z "${time}" && -z "$(which hyperfing 2>/dev/null)" ]]; then
  echo 'time, gtime, or hyperfine must be available in $PATH'
  exit 1
fi
```

Might as well change it for both and exit with error if both are missing and `hyperfine` is missing.

GNU `time` isn't installed by default in e.g. `archlinux:base`. The shell built-in may be available, but that's not the same thing and isn't equivalent. I noticed that today when I started using an Arch container instead of Ubuntu.